pootle/apps/pootle_fs/response.py

```diff
@@ -0,0 +1,139 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+import logging
+from collections import OrderedDict
+
+from django.utils.functional import cached_property
+
+from pootle.core.response import ItemResponse, Response
+from pootle_store.models import Store
+
+
+logger = logging.getLogger(__name__)
+
+FS_ACTION = OrderedDict()
+FS_ACTION["pruned_from_pootle"] = {
+    "title": "Deleted in Pootle",
+    "description":
+        "Stores removed from Pootle that were not present in the filesystem"}
+FS_ACTION["pulled_to_pootle"] = {
+    "title": "Pulled to Pootle",
+    "description":
+        "Stores updated where filesystem version was new or newer"}
+FS_ACTION["added_from_pootle"] = {
+    "title": "Added from Pootle",
+    "description":
+        "Files staged from Pootle that were new or newer than their files"}
+FS_ACTION["pruned_from_fs"] = {
```

Comment:
I know this is an `OrderedDict` but is it no possible to put the `prune*` entries together?